app/lib/methods/search.ts

```diff
@@ -15,10 +15,24 @@ export const localSearchSubscription = async ({ text = '', filterUsers = true, f
 	const searchText = text.trim();
 	const db = database.active;
 	const likeString = sanitizeLikeString(searchText);
+	const slugifiedString = slugifyLikeString(searchText);
 	let subscriptions = await db
 		.get('subscriptions')
 		.query(
-			Q.or(Q.where('name', Q.like(`%${likeString}%`)), Q.where('fname', Q.like(`%${likeString}%`))),
+			Q.or(
+				Q.where('sanitized_fname', Q.like(`%${slugifiedString}%`)),
+				// We still need the others comparisons, because as sanitized_fname is an optional column.
+				// For those rooms that were already on local db the columns will begins as null,
+				// and each interact with the room will sanitize the fname or name and add to the sanitized_fname.
+				// The interacts can be:
+				// - open the room
+				// - receive any updates as changing the name, last message
+				// - join or create a new room
+				// Otherwise, if we want to only search for sanitized_fname, we should clear the local DB
+				Q.where('name', Q.like(`%${likeString}%`)),
+				Q.where('fname', Q.like(`%${likeString}%`)),
+				Q.where('name', Q.like(`%${slugifiedString}%`))
```

Comment:
I didn't get this one either.

Reply:
Because, as the sanitized_fname isn't filled yet, the user could be on a `channel or team`(only those, because the name is slugified by the server), and while the sanitized_fname isn't done, the search could find through the last condition.